src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/Mutect2FilteringEngine.java

```diff
@@ -39,13 +38,13 @@ private void applyContaminationFilter(final M2FiltersArgumentCollection MTFAC, f
         final double alleleFrequency = GATKProtectedVariantContextUtils.getAttributeAsDoubleArray(vc,
                 GATKVCFConstants.POPULATION_AF_VCF_ATTRIBUTE, () -> new double[] {0.0, 0.0}, 0)[maxFractionIndex];
 
-        final double somaticLikelihood = somaticPriorProb / (depth + 1);
+        final double somaticLikelihood = 1.0 / (depth + 1);
 
         final double singleContaminantLikelihood = 2 * alleleFrequency * (1 - alleleFrequency) * MathUtils.binomialProbability(depth, altCount, contamination/2)
```

Comment:
Add the `(1-f)^2`term for when `altCount = 0`

Reply:
Technically true, but if `altCount = 0` there's no variant to filter.

Reply:
Ahhh got it, I missed that